users/autostart.rst

```diff
@@ -20,32 +20,141 @@ there are a number of easy solutions.
+   #. Check "Do not store password. The task will only have
+      access to local resources". Leave this option unchecked if you intend
+      to store your folders on network drives and such.
+
+   |Windows Task Scheduler General Startup Screenshot|
+
+#. Triggers tab:
+
+   #. Click "New...".
+   #. Set "Begin the task:" to "At startup".
+   #. Click "OK".
+
+   |Windows Task Scheduler Triggers Startup Screenshot|
+
+#. Actions tab:
+
+   #. Click "New...".
+   #. Enter the path to ``syncthing.exe`` in "Program/script:" (for example
+      ``C:\syncthing\syncthing.exe``).
+   #. Click "OK".
+
+   |Windows Task Scheduler Actions Startup Screenshot|
+
+#. Settings tab:
+
+   #. Uncheck "Stop task if it runs longer than:".
+   #. Click OK.
+   #. If prompted, enter password for the user.
+
+   |Windows Task Scheduler Settings Screenshot|
+
+Additional configuration in Task Scheduler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following settings are optional. They are not required for Syncthing to
+run properly, but can offer additional functionality. Feel free to make use
+of them depending on your needs.
+
 #. Conditions Tab:
-    #. (optional) Uncheck "Stop if the computer switches to battery power". Note that unchecking "Start the task only if the computer is on AC power" does not disable this option. Even if greyed out, it still applies, and has to be unchecked separately.
-    #. (optional) Uncheck "Start the task only if the computer is on AC power".
-    #. (optional) Check "Start only if the following network connection is available" and set to "Network". Use this option on a laptop, when you want Syncthing to start only on a wired Internet connection, and not on Wi-Fi. Note that once started, Syncthing will not stop running if the connection changes or becomes unavailable later.
-#. Settings Tab:
-    #. Clear checkbox from "Stop task if it runs longer than:".
-#. Click OK.
-#. If prompted, enter password for the user.
+
+   #. Uncheck "Stop if the computer switches to battery power". Note that
+      unchecking "Start the task only if the computer is on AC power" does
+      not disable this option. Even if greyed out, it still applies, and has
+      to be unchecked separately.
+   #. Uncheck "Start the task only if the computer is on AC power".
+   #. Check "Start only if the following network connection is available"
+      and set to "Network". Use this option on a laptop, when you want
+      Syncthing to start only on a wired Internet connection, and not on
+      Wi-Fi. Note that once started, Syncthing will not stop running if the
+      connection changes or becomes unavailable later.
+
+   |Windows Task Scheduler Conditions Screenshot|
+
+#. Actions Tab:
+
+   #. Select the previously created action and click "Edit...".
+   #. Enter the path to the parent folder of ``syncthing.exe`` in "Start in
+      (optional)". This will allow you to use paths relative to this folder
+      in Syncthing.
```

Comment:
What does this do? Respectively where in Syncthing do you want to use relative parts?

Reply:
You can use relative paths for Syncthing folders, e.g. `./Sync` will create a folder `C:\Syncthing\Sync`. Without the "Start in" field defined, the folder will be created in `%SYSTEMROOT%\System32`, so it will be `C:\Windows\System32\Sync`.

Personally, I would say that defining the "Start in" field is a good practice.

Reply:
Just to add, I myself use relative paths in Syncthing, at least on my home computer, where I have a separate `S:` partition specifically to store Syncthing and all my folders there. If something happens, I can then move everything together to a different location, and still be able to run Syncthing only by changing the "Start in" folder, without touching the actual configuration.

Reply:
> It is possible to use relative paths for Syncthing folders, e.g. `./Sync` will create a folder `C:\Syncthing\Sync`. Without the "Start in" field defined, the folder will be created in `%SYSTEMROOT%\System32`, i.e. `C:\Windows\System32\Sync` (which is ... bad).

That's in my opinion a reason to not allow relative paths at all. I think the inconvenience for you to have to type the disk letter of your separate drive does not compare to the weirdness of having a folder in `C:\Windows\System32`. However this hasn't come up as a problem yet, so I am definitely not going to actively destroy the usage scenario of our most active docs contributor ;)

Reply:
I actually agree, but at the same time, this would kill any portable usage of Syncthing, i.e. being able to keep both the installation, and the synced folders on a USB external drive and use it on different machines, so it is a double-edged sword.